src/server/scripts/Spells/spell_rogue.cpp

```diff
@@ -44,6 +44,23 @@ enum RogueSpells
     SPELL_ROGUE_TRICKS_OF_THE_TRADE_PROC        = 59628,
 };
 
+/* Returns true if the spell is a finishing move.
+ * A finishing move is a spell that cost combo points */
+Optional<int32> GetFinishingMoveCPCost(Spell const* spell)
+{
+    if (!spell)
+        return { };
+
+    return spell->GetPowerTypeCostAmount(POWER_COMBO_POINTS);
```

Comment:
GetPowerTypeCostAmount does not exsist in Azerothcore

Reply:
Yes im looking for a fix. If u know a solution. be free :)

Reply:
A solution? You are blindly pushing thru a web browser if I am to understand this right. Please work with it in a IDE so all errors can be viewed and propperly fixed. You are calling for HOOKS that do not EXSIST at all. Implement the HOOKS you are calling for. How were u able to even test this to begin with in your earlier statement if the hooks never exsisted?